### PR TITLE
Update 0.9.0

### PR DESCRIPTION
**Pull request by Arishamays1**
## Scikit-optimize 5.4.2 Update
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-1329)
[Upstream](https://github.com/scikit-optimize/scikit-optimize)
[Dependencies ](https://github.com/scikit-optimize/scikit-optimize/blob/master/setup.py)
# Changes
- Updated version number and `sha256`
- Updated pinnings on dependencies
- Changed build number back to 0
- added `skip: True # [py<36]` to build section

### CODE REVIEW

**Review by sumit0190 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,22 +12,21 @@ source:
...
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
 
 requirements:
   host:
     - python >=3.6
     - pip
```
Comment (katietz) [inline recipe/meta.yaml:22]: missing numpy dependency in host section

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,22 +12,21 @@ source:
...
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
 
 requirements:
   host:
     - python >=3.6
```
Comment (katietz) [inline recipe/meta.yaml]: do not use >=3.6 in host for python.  Instead add a skip block for it ('skip: True. # [py<36]')

**Review by katietz — CHANGES_REQUESTED** [review]

**Comment by katietz** [comment]
Sumit ? how you could approve this?

File: recipe/meta.yaml (hunk 2 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Got it! Went ahead and made this amendment. Thanks!

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml:22]: Added thanks!

**Review by katietz — APPROVED** [review]
thank you, lgtm

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,22 +12,23 @@ source:
...
+  skip: True  # [py<36]
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - numpy {{ numpy }}
   run:
     - python >=3.6
```
Comment (chenghlee) [inline recipe/meta.yaml]: Now that we've dropped `noarch: python`, we don't need `>=3.6` in the `run` section. That limit should be automatically added by the `python` from the `host` section.

**Review by chenghlee — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 3 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Thanks Cheng! I will go ahead and update this. :)

**Pull request closed without merging**